src/components/Modals/Send/hooks/useSendDetails/useSendDetails.tsx

```diff
@@ -317,7 +328,9 @@ export const useSendDetails = (): UseSendDetailsReturnType => {
       'The request failed and the interceptors did not return an alternative response'
     )
       return setValue(SendFormFields.AmountFieldError, 'modals.send.getFeesError')
-    setValue(SendFormFields.AmountFieldError, error?.message ? error.message : '')
+    if (error?.message) {
```

Comment:
It was causing an problem, fee estimation was running before this, errored, but the error was kicked because of this effect while we effectively wanted it

Reply:
Splitted to https://github.com/shapeshift/web/pull/8328